src/components/badges/List.tsx

```diff
@@ -67,33 +39,16 @@ function BadgeListSuspended() {
       <ConfettiIfNeeded />
       <Scrollbar>
         <div className={space('space-y-2')}>
-          <BadgeSection
-            title="Mainnet NFT derivatives"
-            minted={ownedExternalERC721DerivativeContracts}
-            proofs={proofsAvailableToMint.filter(
-              (proof) =>
-                proof instanceof ERC721Proof &&
-                proof.network === Network.Mainnet
-            )}
-            onMinted={onMinted}
-          />
-          <BadgeSection
-            title="Goerli NFT derivatives"
-            minted={ownedERC721DerivativeContracts}
-            proofs={proofsAvailableToMint.filter(
-              (proof) =>
-                proof instanceof ERC721Proof && proof.network === Network.Goerli
-            )}
-            onMinted={onMinted}
-          />
-          <BadgeSection
-            title="Email derivatives"
-            minted={ownedEmailDerivativeContracts}
-            proofs={proofsAvailableToMint.filter(
-              (proof) => proof instanceof EmailProof
-            )}
-            onMinted={onMinted}
-          />
+          {(Object.keys(data) as DataKey[]).map((ledgerName) => (
```

Comment:
Looks like you're doing `Object.keys(data) as DataKey[]` a couple of times, maybe we can put it into `data.ts`:
```TS
export const dataKeys = Object.keys(data) as DataKey[]
```
?

Reply:
You'll make some lines shorter